nexus_zenoh_bridge_dds_vendor/CMakeLists.txt

```
@@ -12,7 +12,7 @@ find_package(ament_cmake_vendor_package REQUIRED)
 
 ament_vendor(zeno_bridge_dds_vendor
   VCS_URL https://github.com/eclipse-zenoh/zenoh-plugin-dds.git
-  VCS_VERSION 0.7.2-rc
+  VCS_VERSION 1.0.0
```

Use the latest patch?

```suggestion
  VCS_VERSION 1.0.2
```